datastore/google/cloud/datastore/helpers.py

```diff
@@ -138,12 +138,12 @@ def entity_from_protobuf(pb):
         if is_list:
             exclude_values = set(value_pb.exclude_from_indexes
                                  for value_pb in value_pb.array_value.values)
-            if len(exclude_values) != 1:
+            if len(exclude_values) > 1:
```

Comment:
@dhermes if `len(exclude_values) == 0`, won't that cause trouble when it *should* have raised? I'm thinking this line should be reverted back?